bot/cogs/alias.py

```diff
@@ -34,19 +34,32 @@ def __init__(self, bot: Bot):
 
         await ctx.invoke(cmd, *args, **kwargs)
 
+    async def test_alias_can_run(self, ctx: Context, alias: str) -> bool:
+        """Check can user use this specific alias."""
+        # We don't want arguments, only command itself and max command parts in aliases is 2.
+        if len(alias.split()) >= 3:
+            alias = " ".join(alias.split()[0:2])
+        # Force allow `get group` due it's not actual command.
+        return alias == "get group" or await self.bot.get_command(alias).can_run(ctx)
```

Comment:
I'm not aware of what `get group` is, this is a fault of my own, but maybe there should be a comment about this in the docstring?
I'm not saying it should be changed, it's more for a discussion than anything else.

Reply:
also, returning both bools is pretty smart. I like it.

Reply:
`get group` is basically `get <group>`, where group can be `tags`, `docs`. Really `tags group` command doesn't exist, just `group` is in the alias function name. I'll add this info to docstring.